sdk/cargo-build-sbf/src/main.rs

```diff
@@ -912,6 +919,17 @@ fn build_solana_package(
     }
 }
 
+// allow user to set proper `rustc` into RUSTC or into PATH
+fn check_solana_target_installed(target: &str) {
+    let rustc = env::var("RUSTC").unwrap_or("rustc".to_owned());
+    let rustc = PathBuf::from(rustc);
+    let output = spawn(&rustc, ["--print", "target-list"], false);
+    if !output.contains(target) {
+        error!("Provided {:?} does not have {} target.", rustc, target,);
```

Comment:
nit: let's add some more info for when someone uses this incorrectly. What do you think of this?
```suggestion
        error!("Provided {:?} does not have {} target. The Solana rustc must be available in $PATH or the $RUSTC environment variable for the build to succeed.", rustc, target,);
```